doc/sphinx/user-extensions/syntax-extensions.rst

```diff
@@ -687,6 +687,31 @@ side. E.g.:
    Notation "'apply_id' f a1 .. an" := (.. (f a1) .. an)
      (at level 10, f ident, a1, an at level 9).
 
+Custom entries
+~~~~~~~~~~~~~~
+
+New grammar entries can be defined using the ``Declare Custom Entry``
```

Comment:
Indeed, I haven't followed these PRs really closely but if the `Declare Custom Entry` command is a new one, it is really important that you present it in the following way:

```
.. cmd:: Declare Custom Entry @ident

   This command allows to define new grammar entries that can later be referred to 
   using the entry name :n:`custom @ident`.

   .. example::

      For instance, we may want to define an *ad hoc* parser for arithmetical operations
      and proceed as follows:

      .. coqtop::
```

In particular, the use of `cmd` allows to find this command in the command index.

If you haven't yet, I suggest you read https://github.com/coq/coq/tree/master/doc/sphinx#documenting-coq-with-sphinx
I find this documentation very helpful to better understand how the new format of the manual works.

Reply:
Hum, I'm very sorry, I had again pushed to a wrong branch name and you did not see the correct version.

Anyway, the correct version had a `.. cmd:: Local Declare Custom Entry @ident` but no `.. cmd:: Declare Custom Entry @ident`, so I added one.

I'm unsure to  have fully understand the difference between single and double backticks. Is it that we use double backticks when in regular text and single ones when argument of a sphinx role/macro? Also, I guess that I should write ``:n:`custom @ident` `` rather than ``` ``custom`` `ident` ``` to mean that I'm thinking at generic code where `ident` is a metavariable, and that I should write ``:g:`custom foo` `` to mean that I'm thinking at a specific example with `foo` taking the place of the metavariable?

My use of ``:g:`...` `` vs ``...`` seems still a bit random.